lib/generators/rolemodel/webpack/webpack_generator.rb

```diff
@@ -2,60 +2,88 @@ module Rolemodel
   class WebpackGenerator < Rails::Generators::Base
     source_root File.expand_path('templates', __dir__)
 
-    def make_project_use_es_modules
+    NODE_VERSION = '18.15.0'.freeze
+
+    DEV_DEPS = %w[
+      @honeybadger-io/webpack
+      @honeybadger-io/js
+      esbuild
+      esbuild-loader
+      webpack
+      webpack-cli
+    ]
+
+    POSTCSS_PKGS = %w[
+      @csstools/postcss-sass
+      postcss
+      postcss-loader
+      postcss-preset-env
+      postcss-scss
+    ]
+
+    WEBPACK_CSS_PKGS = %w[
+      css-loader
+      css-minimizer-webpack-plugin
+      mini-css-extract-plugin
+      webpack-remove-empty-scripts
+    ]
+
+    def ensure_node_version
+      say "Ensuring Node version #{set_color(NODE_VERSION, :yellow)} is installed via nodenv"
+
+      run 'brew update && brew install nodenv node-build'
```

Comment:
This is a bit heavy handed. I know most of us use nodenv, but would want to make this simple for someone using asdf not to get the setup broken.

That's the reason just sticking with the `.node-version` makes sense